NEWS

```diff
@@ -2,6 +2,13 @@ systemd System and Service Manager
 
 CHANGES WITH 243 in spe:
 
+        * systemd-resolved has a "Cache=" option in resolved.conf which
+          has been extended to also accept the 'no-negative' option. Previously
+          only a boolean option was allowed yes/no, having yes as the default.
+          This option skips negative replies from being cached while
+          keeping the same cache heuristics for positive answers.
+          The default remains as "yes" (i. e. caching is enabled).
```

Comment:
there's another item about resolved in the NEWS for 243, please move it close to that

Reply:
Done